code/processes/segmentedtickerplant.q

```diff
@@ -0,0 +1,53 @@
+// Segmented TP process
+// Contains all TP functionality with additional flexibility
+// Configurable logging and subscriptions
+// Default settings create single TP log per table and rolls logs hourly
+// Subscription to a table can be made in two modes - all or filtered
+// All - publish all data for table
+// Filtered - apply filters to published data, filters defined on client side
+
+.proc.loadf[getenv[`KDBCODE],"/common/os.q"];
+
+// Load schema
+$[`schemafile in key .proc.params;
+  .proc.loadf[raze .proc.params[`schemafile],".q"];
+  [.lg.e[`stp;"schema file required"];exit]
+ ]
+
+// Populate pub/sub tables list with schema tables
+.stpps.t:tables[`.]except `currlog;
```

Comment:
Should set namespace with `\d .` like other TorQ processes. See `code/processes/rdb.q`.